internal/cmdlineopt.h

```diff
@@ -26,6 +26,9 @@ typedef struct ruby_cmdline_options {
 #if USE_MJIT
     struct mjit_options mjit;
 #endif
+#if USE_YJIT
+    bool yjit;
+#endif
```

Comment:
Putting a `bool` here would add a padding too.
How about to add after `xflag` or other bit flags?
Furthermore, this can be a bit flag too.

Reply:
Interesting. Fixed it as such at https://github.com/ruby/ruby/pull/6597/commits/9b103e73efb31fa5da1f8b736280a561ba08e431.